conf.py

```diff
@@ -225,6 +225,7 @@ def autoapi_prepare_jinja_env(jinja_env):
                     "ports/nordic/peripherals",
                     "ports/nordic/usb",
                     "ports/raspberrypi/sdk",
+                    "ports/raspberrypi/pioasm",
```

Comment:
Just making sure it does not matter that this is in this PR?

Reply:
It turns out that if you build in ports/raspberry pi and then try to build the docs, it'll show an error because of unexpected files in ports/raspberrpi/pioasm. This is the fix.